python/friesian/example/recall/README.md

```diff
@@ -1,6 +1,112 @@
-https://github.com/facebookresearch/faiss/blob/main/INSTALL.md
+# Offline Recall with Faiss on Spark
+This example demonstrates how to use the retrieval algorithms provided by [faiss](https://github.com/facebookresearch/faiss) 
+to perform the offline recall task efficiently on Spark. 
 
-Need to install via conda to get reasonable performance on CPU:
+
+## Prepare the environment
+We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster (yarn-client mode only).
+
+Note that faiss needs to be installed via conda to get promising performance on CPU. 
+Please refer to [faiss install](https://github.com/facebookresearch/faiss/blob/main/INSTALL.md) for more installation guidance. 
+
+```
+conda create -n bigdl python=3.7  # "bigdl" is the conda environment name, you can use any name you like.
+conda activate bigdl
+conda install -c pytorch faiss-cpu 
+pip install --pre --upgrade bigdl-friesian
+```
+
+## Generate data
+You can generate some test data to run the example, which contains:
+- embeddings: item embeddings of the shape [row_nums, vec_dim].
+- item_dict: unique string names of items with the shape [row_nums, ].
+- index data: the faiss index data generated from item embeddings.
+- parquet data: items saved in parquet format, each row contains two values, the id and the embedding.
```

Comment:
items saved in parquet format to do the search, where each row contains item id and its embedding.